client/src/components/Footer/Footer.css

```diff
@@ -0,0 +1,52 @@
+.footer{
+    background: url("./img/footer-back.jpg");
+    margin:0px auto;
+    padding: 20px 0px 20px 0px;
+    color: #ddd;
+}
+/* House of Geeks logo */
+.logo {
+    height:140px;
+    width:200px;
+}
+.links hr {
+    width:60%;
+}
+#footer-hr { 
+    display: block;
+    width:100%;
+    overflow: hidden;
+    border-width: 1px;
+}
+
+address{
+   font-size:small;
+    margin:0px;
+    color: #ddd;
+}
+#dev {
+    position: absolute;bottom: 0px; left: 0px;
+}
+#up {
+    position: absolute;bottom: 0px; right: 0px;
+}
+
+.footer-a{
+color:#fff;
+}
+
+/* Fontawesome icons styling */
+.iconer {
+    transition: ease 0.5s;
+}
+.iconer:hover {
+    color: #1eabd6;
+    transform: translate(0, -1px);
+}
+@media (max-width:767px){
```

Comment:
So, everything is responsive for the footer?

Reply:
Yes, everything looks responsive , I have tested on all screen width and on Chrome as well as on Mozilla Firefox